tsdb/config.go

```diff
@@ -109,6 +109,7 @@ type Config struct {
 
 	// Options for ingress metrics
 	IngressMetricByMeasurement bool `toml:"ingress-metric-by-measurement-enabled"`
+	IngressMetricByLogin       bool `toml:"ingress-metric-by-login-enabled"`
```

Comment:
Do we want to put some description and examples of this in `etc/config.sample.toml`?

Reply:
Yes that would be a good idea - will fix